pyproject.toml

```diff
@@ -120,4 +99,18 @@ max-statements = 106
 
 [tool.ruff.per-file-ignores]
 "*/__init__.py" = ["F401"]
-"pydicom/config.py" = ["PLW0602"]
+"src/pydicom/config.py" = ["PLW0602"]
+
+[tool.mypy]
+python_version = "3.10"
+files = "src/"
+exclude = ["src/pydicom/benchmarks/", "src/pydicom/pixel_data_handlers/pillow_handler.py"]
+show_error_codes = true
+warn_redundant_casts = true
+warn_unused_ignores = false
```

Comment:
warn_unused_ignores changed to `false` to silence one error, but could put that in the per-file-ignores